x-pack/plugins/observability/public/pages/alerts/alerts_table_t_grid.tsx

```diff
@@ -65,7 +67,7 @@ import { getDefaultCellActions } from './default_cell_actions';
 import { LazyAlertsFlyout } from '../..';
 import { parseAlert } from './parse_alert';
 import { CoreStart } from '../../../../../../src/core/public';
-import { translations } from './translations';
+import { translations, paths } from '../../config';
```

Comment:
I am a bit on the fence on the decision to move `translations` under `config`, but won't block on this 🤔

Reply:
I am also not sure about it but, we can change it any time...